src/test/java/org/elasticsearch/search/highlight/HighlighterSearchTests.java

```diff
@@ -1914,8 +1879,8 @@ public void testPostingsHighlighter() throws Exception {
 
         logger.info("--> searching on _all, highlighting on field1");
         source = searchSource()
-                .query(termQuery("_all", "test"))
-                .highlight(highlight().field("field1").preTags("<xxx>").postTags("</xxx>"));
+                .query(termQuery("field1", "test"))
```

Comment:
I see. Then, I think we should change the log message from `searching on _all` to `searching on field1`